internal/jitsi/jvb.go

```diff
@@ -40,56 +47,100 @@ import (
 )
 
 const (
-	JvbName      = "jvb"
-	externalPort = 10000
+	JvbName         = "jvb"
+	externalPortUDP = 10000
```

Comment:
Does it make sense to move this parameter to some external operator config that can be updated with helm?

Reply:
done